integrations/github/src/haystack_integrations/prompts/github/file_editor_tool.py

```diff
@@ -0,0 +1,120 @@
+file_editor_prompt = """
+Use the file editor to edit an existing file in the repository.
```

Comment:
Fine to ignore this comment, but just so you are aware we are adding an extra new line at the beginning of the prompt by starting the text on the second line. 
```suggestion
file_editor_prompt = """Use the file editor to edit an existing file in the repository.
```

Reply:
Done.